src/deepsparse/server/server.py

```diff
@@ -164,42 +170,54 @@ def _add_endpoint(
     pipeline = Pipeline.from_config(pipeline_config, context)
 
     _LOGGER.info(f"Adding endpoints for '{endpoint_config.name}'")
-    _add_pipeline_endpoint(app, endpoint_config, pipeline, server_config.integration)
+    _add_pipeline_endpoint(
+        app, endpoint_config, pipeline, pipeline_logger, server_config.integration
+    )
 
 
 def _add_pipeline_endpoint(
     app: FastAPI,
     endpoint_config: EndpointConfig,
     pipeline: Pipeline,
+    pipeline_logger: ManagerLogger,
     integration: str = INTEGRATION_LOCAL,
 ):
     input_schema = pipeline.input_schema
     output_schema = pipeline.output_schema
-
-    def _predict_from_schema(request: pipeline.input_schema):
-        return pipeline(request)
+    pipeline_name = pipeline.alias or pipeline.task
+
+    def _predict(request: pipeline.input_schema):
+        (
+            pipeline_outputs,
+            pipeline_inputs,
+            engine_inputs,
+            inference_timing,
+        ) = pipeline.run_with_monitoring(request)
```

Comment:
nit: figure out how to format this on less lines 😁

Reply:
I did try quite a few line breaks, but black keeps forcing back to here, and we've agreed to keep that our source of truth for formatting / breaks.  Don't want to keep the outputs as a tuple to unpack later as there's a more than likely change the outputs of `run_with_monitoring` will change and explicitly naming will make it easier to catch and fix issues when that happens rather than accessing via indexing

Reply:
👍